apps/src/StudioApp.js

```diff
@@ -1323,6 +1323,48 @@ StudioApp.prototype.onMouseMoveVizResizeBar = function (event) {
   this.resizeVisualization(newVizWidth);
 };
 
+StudioApp.prototype.clearVisualizationSize = function () {
+  if ($('#visualizationColumn').hasClass('wireframeShare')) {
+    return;
+  }
+
+  const editorColumn = $(".editor-column");
+  const visualization = document.getElementById('visualization');
+  const visualizationResizeBar = document.getElementById('visualizationResizeBar');
+  const visualizationColumn = document.getElementById('visualizationColumn');
+
+  if (this.isRtl()) {
+    visualizationResizeBar.style.right = '';
+    editorColumn.css('right', '');
+  } else {
+    visualizationResizeBar.style.left = '';
+    editorColumn.css('left', '');
+  }
+
+  visualizationResizeBar.style.lineHeight = '';
+  visualizationColumn.style.maxWidth = '';
+  visualization.style.maxWidth = '';
+  visualization.style.maxHeight = '';
+
+  if (!utils.browserSupportsCssMedia()) {
+    visualization.style.height = '';
+    visualization.style.width = '';
+  }
+
+  applyTransformScaleToChildren(visualization, '');
+
+  $('#soft-buttons').removeClass('soft-buttons-compact');
+
+  const smallFooter = document.querySelector('#page-small-footer .small-footer-base');
+  if (smallFooter) {
+    smallFooter.style.maxWidth = '';
+    const smallPrint = smallFooter.querySelector('small');
+    smallPrint.style.float = '';
+  }
+
+  utils.fireResizeEvent();
```

Comment:
@Bjvanminnen it appears we stopped supporting applab levels of different dimensions besides 320 x 450px 1 year ago: https://github.com/code-dot-org/code-dot-org/pull/7721

This makes me think that now is a good time to remove `adjustAppSizeStyles` since it will greatly simplify the code needed to get thumbnails working. It also makes me think that the remaining work is big enough that I should focus on shipping the rest of Teacher View of Student Projects first.